### ISSUE TITLE
svglib dependency is missing

### ISSUE THREAD

**Issue by MacLake** [SmileyChris/easy-thumbnails#602]
After updating my project with `pip-compile` and `pip-sync` ` easy-thumbnails` is bumped to 2.8.2 and `svglib` removed:

```
diff --git a/requirements.txt b/requirements.txt
[…]
-easy-thumbnails==2.8.1
+easy-thumbnails==2.8.2
     # via
     #   -r requirements.in
     #   aldryn-people
[…]
-svglib==1.3.0
-    # via easy-thumbnails
[…]
```

This leads to an error because in `VIL/Image.py`  there’s an import from `svglib`:

```
  File "/home/jens/.virtualenvs/djangocms3.9/lib/python3.9/site-packages/easy_thumbnails/VIL/Image.py", line 10, in <module>
    from svglib.svglib import svg2rlg
ModuleNotFoundError: No module named 'svglib'
```

Adding `svglib` to `requirements.in` and running `pip-compile && pip-sync` fixes the problem, but the dependencies in `easy-thumbnails` should be fixed.

**Comment by jrief** [comment]
Some users of **easy-thumbnails** complained, that they do not need SVG support and thus don't want to pull in the extra dependency `reportlabs`. Therefore this feature is optional. You can install `pip install easy-thumbnails[svg]` to fetch that extra dependencies.

**Comment by martin056** [comment]
We have the same problem. `reportlab` and `svglib` are missing. My workaround is to add these to the `requirements` file of my project:

```
reportlab==3.6.10
svglib==1.3.0
easy_thumbnails==2.8.2
```

**Comment by jrief** [comment]
Instead you should just add

```
easy-thumbnails[svg]
```

to your `requirements.txt` file.

**Comment by SmileyChris** [comment]
(I'd suggest if we're changing core requirements / making breaking changes in the future, we use a major version bump)

**Comment by derek-adair** [comment]
I'm getting a missing dep error with a fresh install of 2.8.2; `pip install easy-thumbnails` (not using the [svg] option).
```
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'easy_thumbnails.templatetags.easy_thumbnails_tags': No module named 'reportlab'
```
simply trying to render a template manually;

`<img src="{% thumbnail mymodel.image 50x50 crop %}" />`

This is from following the README.

**Comment by MacLake** [comment]
> Some users of **easy-thumbnails** complained, that they do not need SVG support and thus don't want to pull in the extra dependency `reportlabs`. Therefore this feature is optional.

Ok, Jacob, but now even `./manage.py check` fails unless `svglib` is installed because of the import in `VIL/Image.py`.

**Comment by akx** [comment]
My bad for not having properly checked whether everything works without `svglib` installed when implementing #597. Will be hopefully fixed by #603.

**Comment by akx** [comment]
So this should be fixed by upgrading to 2.8.3. Sorry again :)

**Comment by derek-adair** [comment]
Thx budy, much love to the OS maintainers round the world!
